website/content/docs/consul-vs-other/service-mesh-compare.mdx

```diff
@@ -0,0 +1,18 @@
+---
+layout: docs
+page_title: Consul compared to other service meshes
+description: >-
+  Consul's service mesh provides zero trust networking based on service identities to authorize, authenticate, and encrypt network services. Consul's service mesh can also provide advanced traffic management capabilties. Although there are many similar capabilities between Consul and other providers like Istio, Solo, Linkerd, Kong, Tetrate, and AWS App Mesh, we highlight the main differentiating factors for help customers compare. 
+---
+
+# Consul compared to other service mesh software
+
+**Examples**: Istio, Solo Gloo Mesh, Linkerd, Kong/Kuma, AWS App Mesh
+
+Consul’s service mesh allows organizations to securely connect and manage their network services across multiple different environments. Using Envoy as the sidecar proxy attached to every service, Consul ensures that all service-to-service communication is authorized, authenticated, and encrypted. Consul includes traffic management capabilities like load balancing and traffic splitting, which help developers perform canary testing, A/B tests, and blue/green deployments. Consul also includes health check and observability features. 
+
+Consul is platform agnostic and therefore supports any runtime (Kubernetes, EKS, AKS, GKE, VMs, ECS, Lambda, Nomad) and any cloud environment (AWS, Microsoft Azure, GCP, private clouds), making it one of the most flexible service discovery and service mesh platforms. Both the control plane and data plane can run in multiple runtimes with Consul, while other service mesh software provides support for multiple runtimes for the data plane but requires the control plane to run solely on Kubernetes. 
+
+Consul also has several unique integrations with Vault, an industry standard for secrets management. Operators have the option to use Consul’s built-in certificate authority, or they can leverage Vault’s PKI engine to generate and store TLS certificates for both the data plane and control plane. In addition, Consul supports the auto-rotation of TLS certificates on both the data plane and control plane without requiring any type of restarts. This allows for more frequent rotations without incurring additional management burden on operators.
+When deploying Consul on Kubernetes, sensitive data including licenses, ACL tokens, and TLS certificates can be centrally stored on Vault instead of Kubernetes secrets. Vault is much more secure than Kubernetes secrets because it automatically encrypts all data, provides advanced access controls to secrets, and provides centralized governance for all secrets.
```

Comment:
```suggestion
When deploying Consul on Kubernetes, you can store sensitive data including licenses, ACL tokens, and TLS certificates centrally Vault instead of Kubernetes secrets. Vault is much more secure than Kubernetes secrets because it automatically encrypts all data, provides advanced access controls to secrets, and provides centralized governance for all secrets.
```